feature-libs/product-configurator/rulebased/components/attribute/types/base/configurator-attribute-multi-selection-base.component.ts

```diff
@@ -0,0 +1,90 @@
+import { ConfiguratorAttributeBaseComponent } from './configurator-attribute-base.component';
+import { Directive, EventEmitter, Input, Output } from '@angular/core';
+import { Configurator } from '../../../../core/model/configurator.model';
+import { ConfigFormUpdateEvent } from '../../../form/configurator-form.event';
+import { ConfiguratorAttributeQuantityComponentOptions } from '../../quantity/configurator-attribute-quantity.component';
+import { map } from 'rxjs/operators';
+import { BehaviorSubject } from 'rxjs';
+import { ConfiguratorAttributeQuantityService } from '../../quantity/configurator-attribute-quantity.service';
+
+@Directive()
+// eslint-disable-next-line @angular-eslint/directive-class-suffix
+export abstract class ConfiguratorAttributeMultiSelectionBaseComponent extends ConfiguratorAttributeBaseComponent {
+  loading$ = new BehaviorSubject<boolean>(false);
+
+  @Input() attribute: Configurator.Attribute;
+  @Input() ownerKey: string;
+  @Output() selectionChange = new EventEmitter<ConfigFormUpdateEvent>();
+
+  // TODO(#11681): make quantityService a required dependency
+  constructor(
+    protected quantityService?: ConfiguratorAttributeQuantityService
+  ) {
+    super();
+  }
+
+  /**
+   * Checks if we are supposed to render a quantity control on attribute level, which
+   * can be derived from the attribute meta data
+   *
+   * @return {boolean} - Display quantity picker on attribute level?
+   */
+  get withQuantityOnAttributeLevel(): boolean {
+    return (
+      this.quantityService?.withQuantityOnAttributeLevel(this.attribute) ??
+      false
+    );
+  }
+
+  /**
+   * Checks if we are supposed to render a quantity control, which
+   * can be derived from the attribute meta data
+   *
+   * @return {boolean} - Display quantity picker?
+   */
+  get withQuantity(): boolean {
```

Comment:
there is same function in the single base.

Reply:
@AlexanderKuschner, let us have a look at it?

Reply:
As discussed: addressed with #12056